src/common/valgen.cpp

```diff
@@ -499,6 +521,25 @@ bool wxGenericValidator::TransferFromWindow()
             *m_pBool = pControl->GetValue() ;
             return true;
         }
+        else if (m_pInt)
+        {
+            *m_pInt = -1;
+            for (int i = 0 ;
+                pControl ;
+                ++i, pControl = pControl != pControl->GetLastInGroup() ? pControl->GetNextInGroup() : nullptr)
+            {
+                wxCHECK_MSG(!pControl->GetValidator() ||
+                            pControl == pControl->GetFirstInGroup(),
+                        false,
+                        "wxRadioButton group validator must be on first item in group");
+                if (pControl->GetValue())
+                {
+                    *m_pInt = i;
+                }
+            }
+            wxASSERT(*m_pInt >= 0 || !"no selected radio button!");
```

Comment:
We have a more readable form for this:
```suggestion
            wxASSERT_MSG( *m_pInt >= 0, "no selected radio button!" );
```